tests/zquantum/qcbm/ansatz_test.py

```diff
@@ -13,55 +10,52 @@
 
 class TestQCBMAnsatz(AnsatzTests):
     @pytest.fixture
-    def number_of_qubits(self):
+    def n_qubits(self):
         return 4
 
     @pytest.fixture
     def topology(self):
         return "all"
 
     @pytest.fixture
-    def ansatz(self, number_of_qubits, topology):
+    def ansatz(self, n_qubits, topology):
         return QCBMAnsatz(
             number_of_layers=2,
-            number_of_qubits=number_of_qubits,
+            number_of_qubits=n_qubits,
             topology=topology,
         )
 
-    def test_get_executable_circuit_too_many_parameters(
-        self, number_of_qubits, topology
-    ):
+    def test_get_executable_circuit_too_many_parameters(self, n_qubits, topology):
         # Given
         params = [
-            np.ones(2 * number_of_qubits),
-            np.ones(int((number_of_qubits * (number_of_qubits - 1)) / 2)),
-            np.ones(2 * number_of_qubits),
+            np.random.rand(2 * n_qubits),
```

Comment:
Using unseeded random inputs can result in flaky tests with hard to reproduce errors. The new [numpy random](https://numpy.org/doc/stable/reference/random/generator.html) module features a nice locally-seedable generator, WDYT about using it instead?

On the other hand, if you'd like to leverage the randomizer to test a wide range of inputs I'd recommend to use a tool for fuzzy testing like [hypothesis](https://hypothesis.readthedocs.io/en/latest/). One of its features is random seed management, so you can reproduce failing test cases.